### ISSUE TITLE
Forked-Daapd -> Shairport-sync

### ISSUE THREAD

**Issue by aleszczynskig** [owntone/owntone-server#1237]
I am using forked-daapd (latest deb install for raspberry pi) to connect to an instance of shairport-sync also running on a raspberry pi. 

The forked-daapd log is reporting the following continually

Unauthorized request '/ctrl-int/1/getproperty?properties=dmcp.volume' from '2a00:23c7:642f:e001:aa2f:cdb4:ff92:f2bc' (is peer trusted in your config?)

and when I try to control forked-daapd from shairport-sync using dbus, the commands do not work.

I am not sure if this should work or what this message in the log relates too. Can you shed some light on this please?

Many thanks

**Comment by p3ck** [comment]
I don't know if it's the same issue you are seeing but I have noticed that
if my dhcp server is down and I only have an ipv6 address I get this
behavior.  If I refresh the network to get an ipv4 address it works again
because the default ACL allows 192.168.1.0/24 I believe.

**Comment by aleszczynskig** [comment]
Thank you. Of course that is an IPv6 address. It completely escaped me. I will check the relevant configuration files.

**Comment by ejurgensen** [comment]
Yes, to allow requests from this address you need to edit trusted_networks in the config file.

**Comment by aleszczynskig** [comment]
Thank you
